Userland/Applications/Escalator/main.cpp

```diff
@@ -44,14 +44,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     }
 
     auto current_user = TRY(Core::Account::self());
-    auto window = TRY(EscalatorWindow::try_create(executable_path.value(), command, EscalatorWindow::Options { description, current_user, preserve_env, forward_stdin, forward_stdout }));
+    auto window = TRY(Escalator::EscalatorWindow::try_create(executable_path.value(), command, Escalator::EscalatorWindow::Options { description, current_user, preserve_env, forward_stdin, forward_stdout }));
 
     if (current_user.uid() != 0) {
         window->show();
         return app->exec();
-    } else {
-        // Run directly as root if already root uid.
-        TRY(window->execute_command());
-        return 0;
     }
+    // Run directly as root if already root uid.
+    TRY(window->execute_command());
+    return 0;
```

Comment:
Was there any reason behind this change? It looks like a no-op, and it also isn't something that `clang-format` would point out as far as I am aware.

Reply:
Not really, I just thought that the else clause is unnecessary, should I reverse this part?